src/cmap/connection_requester.rs

```diff
@@ -33,27 +32,16 @@ pub(super) struct ConnectionRequester {
 impl ConnectionRequester {
     /// Request a connection from the pool that owns the receiver end of this requester.
     /// Returns None if it takes longer than wait_queue_timeout before the pool returns a result.
```

Comment:
Can you remove the second line from this comment?

Reply:
Good catch, done.